ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/routing.j2

```diff
@@ -43,6 +44,8 @@
 {%     include 'documentation/router-service-insertion.j2' %}
 {## Router Traffic Engineering #}
 {%     include 'documentation/router-traffic-engineering.j2' %}
+{## Router Adaptive Virtual Topology #}
+{%     include 'documentation/router-adaptive-virtual-topology.j2' %}
```

Comment:
Should it be "somehow" in alphabetical order regarding the `router-FOO` documentation sections?

Reply:
addressed in 94ffac7